### PR TITLE
Add query param to fabric-website/index.html to check for v5 vs v6.

### PR DESCRIPTION
**Pull request by leddie24**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ npm run change`

#### Description of changes

Adds query parameter to check for fabric query parameter. 

#### Focus areas to test

(optional)

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/8343)

### CODE REVIEW

File: apps/fabric-website/index.html [hunk 1]
```diff
@@ -57,7 +60,9 @@
 
     // If the site is hosted, load the flight config script for production or dogfood
     if (!isLocal) {
-      var configPrefix = isProduction ? 'fabric-website-prod' : 'fabric-website-df';
+      var v5 = fabricVersion == '5' ? 'v5-' : '';
```
Comment (KevinTCoughlin) [inline apps/fabric-website/index.html]:
```suggestion
      var v5 = fabricVersion == '5' ? 'v5-' : '';
```

Prefer `===` unless there's a reason not to.

**Comment by leddie24** [comment]
> This'll work for now, but you might consider making this a more robust check for the upcoming `6` vs. `7` distinction.
> 
> i.e. if query param present, use supported version, else default to latest.

I'm assuming once we have v6, it'll just be `fabric-website-v6-df` (or prod) and 7 will default to `fabric-website-df`?

File: apps/fabric-website/index.html [hunk 2]
```diff
@@ -57,7 +60,9 @@
 
     // If the site is hosted, load the flight config script for production or dogfood
     if (!isLocal) {
-      var configPrefix = isProduction ? 'fabric-website-prod' : 'fabric-website-df';
+      var v5 = fabricVersion === '5' ? 'v5-' : '';
```
Comment (Jahnp) [inline apps/fabric-website/index.html]: Agreed with @KevinTCoughlin's callout. @leddie24, this check could be made more durable--for example, have something like a map/array of supported versions and check if the query param is in it. If yes, use it; if not, fall back to latest.

Comment (leddie24) [reply apps/fabric-website/index.html]: Added an array with supported versions.   If the support version exists in the array and isn't the latest version, then it adds the prefix modifier to the `configPrefix`.

**Pull request merged**